tg/update_handlers.py

```diff
@@ -0,0 +1,235 @@
+import logging
+from functools import wraps
+from typing import Any, Callable, Dict, Optional
+
+from tg import config
+from tg.controllers import Controller
+from tg.msg import MsgProxy
+
+log = logging.getLogger(__name__)
+
+_update_handler_type = Callable[[Controller, Dict[str, Any]], None]
+
+handlers: Dict[str, _update_handler_type] = {}
+
+
+def update_handler(update_type):
```

Comment:
cools like an inconsistency. 
for keybindings you are manually registering shortcut handlers in the dict, but here you are using decorator to register it. Maybe stick to one approach?

Reply:
Truly saying, I am not sure what is the best way to make it readable.

As for me, I prefer seeing keybinding in one place though I don't care about update_hanlders.

What's your propositions? Which way you prefer?